pkg/didcomm/protocol/didexchange/service_test.go

```diff
@@ -20,7 +20,7 @@ import (
 	"github.com/hyperledger/aries-framework-go/pkg/didcomm/dispatcher"
 	"github.com/hyperledger/aries-framework-go/pkg/didcomm/protocol/decorator"
 	"github.com/hyperledger/aries-framework-go/pkg/didcomm/transport"
-	mocktransport "github.com/hyperledger/aries-framework-go/pkg/internal/didcomm/transport/mock"
```

Comment:
do we really want all mocks in a single folder?

Reply:
it's easier to find in one folder

Reply:
Well, we can argue on both sides though. Personally, i feel folder structure becomes too complicated if we have a folder for each mock. anyways, the mocks are inside internal folder.

Reply:
even if you don’t want one folder per package, are we sure we shouldn’t separate mocks at the top level … e.g., didcomm

Reply:
I think we should go with  pkg/internal/mocks/didcomm

Reply:
@troyronda @fqutishat may be going down one level makes sense, i guess.

reg name - i feel putting mock at higher level than feature will give a good readability. People can go to mocks folder and see variable feature level mocks available.

Reply:
[Updated](https://github.com/hyperledger/aries-framework-go/compare/257b15506fd69db7afc2e44467e044d9f3af5a18..9e430efccae39bf51273eafe671bffd5120dec70)